.github/workflows/build-docker-images.yml

```
@@ -13,6 +13,6 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v3
-    - name: Build the full Docker image
-      run: docker build . --file Dockerfile --tag streamlitapp:latest
\ No newline at end of file
+      - uses: actions/checkout@v3
+      - name: Build the full Docker image
+        run: docker build . --file Dockerfile --tag streamlitapp:latest -output type=tar,dest=/dev/null
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)